code/game/atoms_movable.dm

```diff
@@ -917,12 +928,13 @@
 
 /atom/movable/proc/doMove(atom/destination)
 	. = FALSE
-	move_stacks++
 	var/atom/oldloc = loc
+	var/is_multi_tile = bound_width > 32 || bound_height > 32
```

Comment:
Magic numbers, we've got a define for the icon size we're using, please use that instead in the hypothetical future that we change our default icon size.

Reply:
done'd